bastionzero/service/targets/database.go

```diff
@@ -14,6 +14,26 @@ const (
 	databaseSinglePath = databaseBasePath + "/%s"
 )
 
+// CreateDatabaseTargetRequest is used to create a new Database target
+type CreateDatabaseTargetRequest struct {
+	TargetName      string `json:"targetName"`
+	ProxyTargetID   string `json:"proxyTargetId"`
+	RemoteHost      string `json:"remoteHost"`
+	RemotePort      Port   `json:"remotePort"`
+	LocalPort       *Port  `json:"localPort,omitempty"`
+	LocalHost       string `json:"localHost,omitempty"`
+	IsSplitCert     bool   `json:"splitCert,omitempty"`
+	DatabaseType    string `json:"databaseType,omitempty"`
+	EnvironmentID   string `json:"environmentId,omitempty"`
+	EnvironmentName string `json:"environmentName,omitempty"`
+}
+
+// CreateDatabaseTargetRequest is the response returned if a Database target is
+// successfully created
+type CreateDatabaseTargetResponse struct {
```

Comment:
Typo in comment: `CreateDatabaseTargetRequest` --> `CreateDatabaseTargetResponse`